requirements-dev.txt

```diff
@@ -5,3 +5,4 @@ pytest>=7.1.1
 pytest-cov>=3.0.0
 python-dotenv>=0.20.0
 responses>=0.20.0
+tox>=3.25.1
```

Comment:
generally this is too late and will install tox inside tox which isn't what you want.  the usual approach for this is to have tox ~somewhat globally available